wolox-books/src/app/screens/auth/auth.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="main-container">
   <header class="navbar">
     <div class="container navbar-interior">
-      <img class="wolox-logo" src="/assets/wolox_books_logo.png" alt="wolox-logo">
+      <img class="wolox-logo" src="/assets/wolox_books_logo.png" alt="wolox-logo" (click)="goToBooks()">
```

Comment:
wrapping the img with an `a` tag is better for this case. since it's more semantic also using `routerLink="/books"` is a better approach than calling a controller function